File: src/Uri/_2_August/URI_1078_MultiplicationTable.java
package Uri._2_August;

import java.util.Scanner;

public class URI_1078_MultiplicationTable {
    public static void main(String[] args) {
        int n, mul;
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();

        for (int i = 1; i <= 10; i++) {
            System.out.printf(i + " x " + n + " = " + i * n);
        }
    }
}

File: src/Uri/_2_August/URI_1078_MultiplicationTable.kt
package Uri._2_August

import java.util.*

fun main(args:Array<String>){

    var n:Int;
    var mul:Int;
    var sc = Scanner(System.`in`)
    n = sc.nextInt();
    for (i in 1..10){
        mul = i * n;
        println("$i x $n = $mul")
    }
}